clang/lib/CIR/CodeGen/CIRGenExprConst.cpp

```diff
@@ -43,6 +43,19 @@ using namespace clang::CIRGen;
 namespace {
 class ConstExprEmitter;
 
+mlir::TypedAttr getPadding(CIRGenModule &CGM, CharUnits size) {
```

Comment:
To make this easier to read, can you rename it to `computePadding`? You should also make it `static`.